spring-cloud-skipper-server/pom.xml

```diff
@@ -53,4 +57,43 @@
             </plugin>
         </plugins>
     </build>
+
+	<profiles>
+		<profile>
+			<id>docker</id>
+
```

Comment:
do we want this as a profile or just use `./mvnw package dockerfile:build` without adding the plugin to the package phase

Reply:
Probably just fine with `dockerfile:build` just to be consistent with other projects that generate docker images.